packages/app/background-jobs-common/src/background-job-processor/utils.ts

```diff
@@ -15,6 +15,12 @@ export const resolveJobId = (job?: SafeJob<unknown>): string => job?.id ?? 'unkn
 export const isUnrecoverableJobError = (error: Error): boolean =>
   error.name === 'UnrecoverableError'
 
+export const MUTED_UNRECOVERABLE_ERROR_SYMBOL = Symbol.for('MUTED_UNRECOVERABLE_ERROR_KEY')
```

Comment:
would be useful to include a comment, explaining its purpose.

can you also mention this in readme?

Reply:
Done in https://github.com/lokalise/shared-ts-libs/pull/584/commits/188f61b8fbe07a0dbb9953e087b22f785cac5e26